### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by kcotrinam**
**In this project I:**
- Have used html to create a basic template.
- Have used Scss to style it.
- User OOP in order to create the card which display the information coming from the API.
- I have used webpack in order to output the Javascript, CSS and HTML files.
- User ope weather API in order to get the information the city climate.

### CODE REVIEW

File: src/modules/request.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+const apiKey = '90389a28c75f30a2126f4ec7e1c08520';
+
+const getData = async (city) => {
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}`);
```
Comment (Forison) [inline src/modules/request.js]: let us use error handlers for our promises

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::changes required❌
# 👋 Kevin
#### You have done a good job on this one so far, Awesome pull request and commit messages. Code is well organized and easy to read, however here are few issues we have to address to get this project approved🤗.

* [x] Let us complete the README's get started instruction. Now it only ends at cloning the repo, let us include steps to launch the application.

* [x]  The README should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary).
you are not required to expose the key itself.

* [x] It changes its appearance based on the received data ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
It has error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

* [x] It has error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
let us use error handling block like `try and catch` to handle the API request.

#### That is it kindly make the change and submit a re-review👷.

#### Happy coding👨‍💻

Addo Forison

File: src/modules/request.js (hunk 1 above)
Comment (kcotrinam) [reply src/modules/request.js]: thanks for your review

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required :o: 
Great job implementing the required changes.

As mentioned in the last review, API keys should not be exposed to the version control system.

**After making the changes, submit another request for review.**

**Happy Coding :computer:**

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @kcotrinam :wave: 

Your project is coming along just fine, but your API is still exposed. I took a snapshot from your repository on github.

api

Here is what I recommend, learn how to use **environment variables**, in a react/JS projects. 
To prevent committing/exposing your API to your GitHub repository, **create a .env file** and add the API key there. Then add the .env file to .gitignore file.

#### That should do it, when you are done, submit another form for a code review.

### Have a great day :slightly_smiling_face: 

### ``` Uchechukwu Henry Inyama ```

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi @kcotrinam! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**